### PR TITLE
[NLU-4070] Unskip Date Time tests - 5 languages

### PR DESCRIPTION
**Pull request by KanchanKumar12**
Unskip Date Time tests - 5 languages (Arabic, Catalan, Dutch, English, EnglishOthers)

Pipeline run: https://jenkins.ininica.com/job/NLU/job/Recognizers-Text/414/console

### CODE REVIEW

File: Specs/DateTime/Arabic/DatePeriodParser.json [hunk 1]
```diff
@@ -275,7 +275,7 @@
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
     "IgnoreResolution": "true",
```
Comment (andrew-gradinari) [inline Specs/DateTime/Arabic/DatePeriodParser.json:277]: I'm pretty sure the `IgnoreResolution` flag doesn't have any effect on the Python tests. But since we no longer sync with the upstream repository it might be a good idea to remove this flag to avoid any confusion. We can do this in a separate PR though.

Comment (KanchanKumar12) [reply Specs/DateTime/Arabic/DatePeriodParser.json:277]: Took a note of it. Will make changes in the next PR. Thanks Andrew 👍

**Review by andrew-gradinari — APPROVED** [review]

File: Specs/DateTime/Arabic/DatePeriodParser.json (hunk 1 above)
Comment (andrew-gradinari) [reply Specs/DateTime/Arabic/DatePeriodParser.json:277]: Thank you!

**Review by colm-dillon — APPROVED** [review]

**Review by BobbyMartins5 — APPROVED** [review]
